docs/overview.yaml

```diff
@@ -414,11 +414,41 @@ Geometry:
   Differentiable manifolds:
     smooth or analytic manifold (with boundary and corners): 'IsManifold'
     smooth map between manifolds: 'ContMDiff'
+    product of manifolds: 'ModelWithCorners.prod'
```

Comment:
`ModelWithCorners.prod` is not really the product of two manifolds, it's just the corresponding model with corner. Maybe mention instead the instance that a product of smooth manifolds is a smooth manifold? Same thing with the next two entries.

Reply:
I disagree about interior and boundary, though: if you look at the declarations, you see that they *are* about the interior and boundary. (I'm happy to rename them.)